proof_of_concept/ddm_client.py

```diff
@@ -56,10 +52,10 @@ def __init__(self, site: str, site_validator: Validator) -> None:
 
         registry_validator = Validator(registry_api_def)
 
-        registry_client = RegistryClient(
+        registry_client = RegistryRestClient(
```

Comment:
```suggestion
        registry_rest_client = RegistryRestClient(
```
To make it less confusing.

Reply:
I agree, but can we do this as part of #33? I think it makes more sense there.

Reply:
Sure!